cmd/manager/chartrepos.go

```diff
@@ -58,6 +58,14 @@ func addChartRepoHandlerFunc(w http.ResponseWriter, r *http.Request, c *router.C
 		return nil
 	}
 
+	if string(cr.Format) == "" {
+		cr.Type = repo.GCSRepoType
```

Comment:
I dunno about the context of this code but this looks suspiciously like a copy/paste error from below